OnShift/SendBird-JavaScript
# Pl 67 clear out search button

# Things Done
- add functionality which clears search field if clicked
  - if content exists in search bar, icon changes to 'X' from the magnifying glass
  - when clicked, clears the search bar and also changes the icon back to the magnifying glass
- update README
- better build process
  - can only process SVG images
  - utilize better loader for web config
- cache certain images in order to limit network queries (specifically the 'X' icon)
- minor CSS changes

# How To Test
- spin up the widget in both mobile and desktop views
- do both: 
  - create new chat, search for users, press the clear, ensure input is cleared out
  - inviting existing users to chat, press the clear, ensure input is cleared
